tests/pytests/test_vecsim.py

```diff
@@ -1751,6 +1757,17 @@ def test_hnsw(self):
 
         self.run_long_queries(n_vec, query_vec)
 
+    # TODO: fix, failed right now
+    def test_svs(self):
+        # Create index and load vectors.
+        n_vec = self.index_sizes['SVS-VAMANA']
+        query_vec = load_vectors_to_redis(self.env, n_vec, 0, self.dim, self.type)
+        self.env.expect('FT.CREATE', 'idx', 'SCHEMA', 'vector', 'VECTOR', 'SVS-VAMANA', '6', 'TYPE', self.type,
+                        'DIM', self.dim, 'DISTANCE_METRIC', 'L2').ok()
+        waitForIndex(self.env, 'idx')
+
+        self.run_long_queries(n_vec, query_vec)
+
```

Comment:
Note that svs is not supported yet with multi (WIP), so `test_create_multi_value_json` cannot pass. In the meantime you can add in line 1785 :
```
#todo: enable when multi is supported
if algo == 'SVS-VAMANA':
    continue
```